src/click/types.py

```diff
@@ -6,6 +6,7 @@
 import stat
 import sys
 import typing as t
+from configparser import RawConfigParser
```

Comment:
Don't pull in `configparser` for this, it implies a level of support/parity that I don't think is intended.

Reply:
Fixed in https://github.com/pallets/click/pull/2956/commits/2f794921bdb8e65b095e6b000a7955822be38a64